benchflow-dsl/src/main/scala/cloud/benchflow/dsl/explorationspace/ExplorationSpaceGenerator.scala

```diff
@@ -0,0 +1,126 @@
+package cloud.benchflow.dsl.explorationspace
+
+import cloud.benchflow.dsl.definition.BenchFlowTest
+import cloud.benchflow.dsl.definition.types.bytes.Bytes
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-25
+ */
+object ExplorationSpaceGenerator {
```

Comment:
I need comments on this class to understand what is the algorithm you implement.